foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/enable-habushu-build-cache-migration.feature

```diff
@@ -0,0 +1,17 @@
+@habushu-build-cache-migration
+Feature: Adds the <build><directory>dist</directory></build> tag to the pom file if not present.
+
+  Scenario: Migrate a pom file
+    Given a habushu-packaged pom file where the build directory is unspecified
+    When the migration executes
+    Then the pom file contains a specification of the build directory
+
+  Scenario: A pom file is not packaged by habushu
+    Given a non-habushu-packaged pom file where the packaging is unspecified
+    When the migration executes
+    Then the pom file will not be marked for migration
+
+  Scenario: A pom file already specifies the build directory
```

Comment:
S: habushu packaged pom file already contains a build directory.

S: Here - should we ware the user that it will require a custom cache configuration if not aligned to `dist`?

Reply:
Improved the scenario name.

I think a good spot for the warning is actually the release notes, rather than the baton migration, where it can get buried in the output.